### ISSUE TITLE
change repo1-s3-endpoint 

### ISSUE THREAD

**Issue by deanpeterson** [CrunchyData/postgres-operator#3273]
I want to change my repo1-s3-endpoint in the config map created by the operator but there is a warning that my changes won't be saved. Other than destroying my database and creating a new one with the operator is there another way to modify the repo1-s3-endpoint? 

Environment:
CrunchyDB Operator with postgresql 13
Openshift: 4.9.23

**Comment by deanpeterson** [comment]
Figured it out. I shut down the PGO pod temporarily to make a change to the yaml that then propagated to the configmap

**Comment by cku-github** [comment]
I had the a very similar issue on a Test DB.
However I never even setup backup making things worse.
Manually deleting WAL using 
pg_archivecleanup /pgdata/pg13_wal 000000040000000E000000F6
from an openshift terminal to the database instance in the Primary POD, resolved the disk space issue, but as I deleted too many WAL, I managed to brick the installation :(
The secondary responded it was unhealthy as did the Primary.
So this is NOT recommended :)
